src/Button/ZoomToExtentButton/ZoomToExtentButton.jsx

```diff
@@ -40,16 +40,18 @@ class ZoomToExtentButton extends React.Component {
     map: PropTypes.instanceOf(OlMap).isRequired,
 
     /**
-     * The array extent[minx, miny, maxx, maxy]  (the values must be in the maps coordination system) or instance of Ol SimpleGeometry that the map should zoom to.
-     * @type {Array}
+     * The extent `[minx, miny, maxx, maxy]` in the maps coordinate system or an
+     * instance of Ol SimpleGeometry that the map should zoom to.
+     * @type {Array<Number>|OlSimpleGeometry}
      */
     extent: PropTypes.oneOfType([
       PropTypes.arrayOf(PropTypes.number),
       PropTypes.instanceOf(OlSimpleGeometry)
     ]).isRequired,
 
     /**
-     * Options for fitting to the given extent. See https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#fit
+     * Options for fitting to the given extent. See
+     * https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#fit
      * @type {Object}
      */
     fitOptions: PropTypes.shape({
```

Comment:
Not related to the changes introduces in this PR, but the prop `fitOptions` doesn't reflect the current state of the supported options by ol. In order to become more flexible here, I would suggest to set the typing to `PropTypes.object`.

Reply:
:+1: